main.py

```diff
@@ -1,9 +1,20 @@
+import io
+
+from rest_framework.parsers import JSONParser
+from rest_framework.renderers import JSONRenderer
+
 from car.models import Car
+from car.serializers import CarSerializer
 
 
 def serialize_car_object(car: Car) -> bytes:
-    pass
+    return JSONRenderer().render(CarSerializer(car).data)
 
 
 def deserialize_car_object(json: bytes) -> Car:
-    pass
+    stream = io.BytesIO(json)
+    data = JSONParser().parse(stream)
+    serializer = CarSerializer(data=data)
+    if serializer.is_valid():
+        return serializer.save()
+    return serializer.errors
```

Comment:
use is_valid(raise_exception=true) to do all checks automatically and raise handled errors if it's needed